less/index.less

```diff
@@ -57,3 +57,285 @@ html, body {
     height: 100%;
 }
 
+@header-font: 'Lobster', cursive;
+@fav-color:green;
+
+.header(){
+	font-family: @header-font;
+	font-size: 4rem;
+	
+}
+
+.subhead(){
+	font-size: 1.7rem;
+}
+
+.flex-column(){
+	display: flex;
+	flex-direction: column;
+}
+
+.historypics(){
+	width: 25%;
+	margin: 1%;
+
+	img{
+	width: 90%;
+	}
+}
+
+.mobhist(){
+	@media screen and (max-width: 500px){
+		width: 90%;
+		margin: 2%;
+		align-self: center;
+
+		img{
+			width: 90%;
+		}
+	}
+}
+
+
+h1 {
+	.header();
+}
+body{
+
+
+.container {
+	@media screen and (max-width: 500px){
```

Comment:
I think you may be nesting a bit too deep. I personally wouldn't want to nest more than three deep. You don't need to nest global styles either. Yes, it is syntactically correct and you could nest EVERYTHING inside your container, buuuutttt this actually makes it harder for other developers to follow along your code.

Reply:
Thanks for the advice!  I got so into the coding, I forgot not to nest past grandchildren.  I'll keep a closer eye out in the future!